gradle/enforcement/spotless.license.java

```diff
@@ -13,3 +13,4 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
```

Comment:
Strange. There is new line at the end of this file in my clone...

Reply:
Yeah spotless seems to ignore one trailing line so adding a second did the trick